src/PwdResetEmailSentController.js

```diff
@@ -39,7 +39,7 @@ function (Okta, Enums, FormController, FormType) {
       noButtonBar: true,
       attributes: { 'data-se': 'pwd-reset-email-sent' },
       formChildren: function () {
-        return [
+        let views = [
```

Comment:
nit: rename `views` to `children` to be consistent with other similar controllers